test/webauthn/wasm_exec.js

```diff
@@ -0,0 +1,639 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
Why do we need to use wasm here? Are there no libraries that'd do this natively or abstract this away?